### ISSUE TITLE
Feature request: NVDA should announce when something has changed on the screen

### ISSUE THREAD

**Issue by haruncetinkaya306** [nvaccess/nvda#17702]
### Is your feature request related to a problem? Please explain.
On some websites, when I click something, something actually changes on the screen. But I can imagine that I can't click because the page doesn't refresh or NVDA doesn't respond at all. However, when I move with the down arrow key, I see that something has changed there.
### Explain your desired solution
In such cases, NVDA should somehow notify me audibly that the screen has changed.
### Explain the alternatives you are considering
NVDA can also provide this feedback by playing a sound.
### Additional context
None.

**Comment by hwf1324** [comment]
Related: #17444

**Comment by SaschaCowley** [comment]
Closing in favour of #498.
